src/test/kotlin/day03/HeroTest.kt

```
@@ -1,29 +1,33 @@
 
-@org.junit.Before
- fun setUp() {}
-  // 테스트 시랭 전에 수행되는 부분
+class HeroTest {
 
-@org.junit.After
- fun tearDown() {}
-  // 테스트 수행 후에 수행되는 부분
+    @org.junit.Before
+    fun setUp() {
+    }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)